app/Notifications/Training/RemovedFromWaitingListInactiveAccount.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Notifications\Training;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+
+class RemovedFromWaitingListInactiveAccount extends Notification
+{
+    use Queueable;
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param  mixed  $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        return ['mail'];
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     *
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        return (new MailMessage)
+        ->from('support@vatsim.uk', 'VATSIM UK - Member Services')
+        ->subject('UK ATC training waiting list removal')
```

Comment:
I believe we have pilot training lists too, perhaps this should be
```suggestion
        ->subject('UK training waiting list removal')
```

Reply:
Title Case Feels Appropriate For A Subject Too

Reply:
Amended as suggested.